.circleci/config.yml

```diff
@@ -35,13 +35,13 @@ jobs:
           GOOS: linux
```

Comment:
Ideally we would move this over to GitHub Actions but we can handle that in a later PR.